pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dao/JdbiFormSectionBlock.java

```diff
@@ -59,7 +56,13 @@ long[] bulkInsert(@BindMethods("dto") List<SectionBlockMembershipDto> membership
             + " where form_section__block_id = :id")
     int[] bulkUpdateRevisionIdsByIds(@Bind("id") List<Long> membershipIds,
                                      @Bind("revisionId") long[] revisionIds);
-
+    
+    @UseStringTemplateSqlLocator
+    @SqlQuery("queryOrderedFormBlockDtosBySectionIdsAndInstanceGuid")
+    @RegisterConstructorMapper(value = FormBlockDto.class)
```

Comment:
`value = ` might be avoided since FormBlockDto.class is the only one argument here